source/P0323R9.bs

```diff
@@ -838,13 +839,15 @@ template<class G = E>
     <td>`!bool(*this)`</td>
   </tr>
   <tr>
-    <td>assigns `unexpected(e.error())` to `unex`</td>
 <td>
 
+  * move constructs an `unexpected<E> tmp` from `e.value()` (which may throw)
```

Comment:
Temporary `unexpected<E>` should be created from `unexpected<G>` before destroying `val` to ensure strong exception guarantee.